storage/Utils/AsciiFile.cc

```diff
@@ -114,16 +120,23 @@ namespace storage
 	{
 	    y2mil("saving file " << name);
 
-	    ofstream file(name);
-	    classic(file);
+	    FILE* file = fdopen(open(name.c_str(), O_WRONLY | O_TRUNC | O_CREAT, permissions), "w");
+
+	    if (!file)
+		ST_THROW(IOException(sformat("Opening file %s failed: %s", name, strerror(errno))));
 
 	    for (const string& line : lines)
-		file << line << '\n';
+	    {
+		string line_with_break = line + "\n";
 
-	    file.close();
+		fputs(line_with_break.c_str(), file);
+
+		if (ferror(file))
+		    ST_THROW(IOException(sformat("Saving file %s failed: %s", name, strerror(errno))));
+	    }
 
-	    if (!file.good())
-		ST_THROW(IOException(sformat("Saving file %s failed.", name)));
+	    if(fclose(file) == EOF)
+		y2war("closing file " << name << "failed");
```

Comment:
No, failure to write a file like this must be reported to the user. See bsc #1115801.